### DISCUSSION TITLE
Stumped

### DISCUSSION THREAD

**Discussion by rhazzed** [riffnshred/nhl-led-scoreboard#213] [General]
Everytime I unplug my board  it shuts off. Any idea what would caise that?

**Comment by rhazzed** [comment]
Do you think plugging it back in will help?

**Reply by ebrucker406** [reply]
It might, have you tried turning it off and back on?

**Reply by rhazzed** [reply]
Thanks!  turning it back on apparently fixes it

**Comment by riffnshred** [comment]
Good first post. XD
